source/MainWindow.cpp

```diff
@@ -368,7 +368,7 @@ void MainWindow::GenerateGrid(uint8 size)
 	fGrid->GeneratePuzzle();
 	fWorkGrid = new Grid(size);
 
-	BPoint origin = BPoint(0,0);
+	BPoint origin = BPoint(-100,-100);
```

Comment:
Have you tested the game with this change to make sure it doesn't have any unintended results?